packages/shared/src/open-feature.ts

```diff
@@ -16,8 +16,9 @@ export abstract class OpenFeatureCommonAPI<P extends CommonProvider = CommonProv
   protected abstract _defaultProvider: P;
   protected abstract readonly _events: GenericEventEmitter;
 
+  protected readonly _defaultContext = {};
   protected _hooks: H[] = [];
-  protected _context: EvaluationContext = {};
+  protected _context: EvaluationContext = this._defaultContext;
```

Comment:
I might be better to leave this out since it can probably be done with just `{}`, and that way you've isolated your changes to just the web-sdk, but that's minor.

Reply:
Yeah, I had the same concern. I'll remove this.